src/index.ts

```diff
@@ -181,66 +184,66 @@ function startApp(appSchema, path: string) {
     server.applyMiddleware({ app, path })
   } else {
     const graphqlHTTP = require("express-graphql")
-    app.use(
-      graphqlHTTP((req, res, params) => {
-        const accessToken = req.headers["x-access-token"] as string | undefined
-        const appToken = req.headers["x-xapp-token"] as string | undefined
-        const userID = req.headers["x-user-id"] as string | undefined
-        const timezone = req.headers["x-timezone"] as string | undefined
-        const userAgent = req.headers["user-agent"]
-
-        const { requestIDs } = res.locals
-        const requestID = requestIDs.requestID
-
-        if (enableRequestLogging) {
-          fetchLoggerSetup(requestID)
-        }
-
-        // Accepts a tz database timezone string. See http://www.iana.org/time-zones,
-        // https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
-        let defaultTimezone
-        if (timezone && moment.tz.zone(timezone)) {
-          defaultTimezone = timezone
-        }
-
-        const loaders = createLoaders(accessToken, userID, {
-          requestIDs,
-          userAgent,
-          appToken,
-        })
-
-        const context: ResolverContext = {
-          accessToken,
-          userID,
-          defaultTimezone,
-          ...loaders,
-          // For stitching purposes
-          exchangeSchema,
-          requestIDs,
-          userAgent,
-        }
+    const graphqlServer = graphqlHTTP((req, res, params) => {
+      const accessToken = req.headers["x-access-token"] as string | undefined
```

Comment:
#sominor Maybe store type info in a type var? 
```
type Header = string | undefined 
```

Reply:
Oh I concur, tho I'd rather not refactor any of this for the moment 🗡️